src/main/java/uk/gov/ch/service/payment/impl/ConfirmationStatementPaymentCheckServiceImpl.java

```diff
@@ -36,15 +36,15 @@ public ConfirmationStatementPaymentJson isConfirmationStatementPaid(String compa
             Long paidByTransactionId = confirmationStatementPayment.get().getPaidByTransactionId();
 
             if(paidByTransactionId != null) {
-                LOGGER.info("Confirmation statement payment id " +
-                        paidByTransactionId + " found for due date " + dueDateString);
+                LOGGER.info(String.format("Company Number: %s Confirmation statement payment id %s found for due date %s",
```

Comment:
need to change references to 'due date' to 'made up to date' including variable names